cypress/support/step_definitions/testSetup.js

```diff
@@ -55,6 +55,21 @@ Given('the environment has been set up for {string} tests', (test) => {
     if (Cypress.env(CONSTANTS.TEST_TYPE).includes('rpc-Only')) {
       Cypress.env(CONSTANTS.IS_RPC_ONLY, true);
     }
+    // fetch device details dynamically
```

Comment:
this should be a list as mentioned yesterday

Reply:
updated as a list